creator-node/src/services/sync/processSync.js

```diff
@@ -332,6 +332,9 @@ async function processSync (serviceRegistry, walletPublicKeys, creatorNodeEndpoi
         await redisLock.removeLock(redisKey)
 
         logger.info(redisKey, `Transaction successfully committed for cnodeUser wallet ${fetchedWalletPublicKey}`)
+
+        // track that sync for this user was successful
+        await SyncHistoryAggregator.recordSyncSuccess(fetchedWalletPublicKey)
```

Comment:
why move this call? seems functionally same right - except with your change, we could record a sync success for a wallet multiple times if there are multiple associated cnodeUsers?

Reply:
i need to do this to capture the walletPublicKey cause we have a for loop over the array of wallets

Reply:
oh duh lol